packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx

```diff
@@ -0,0 +1,179 @@
+    [arbiterName, setArbiterName, 'Arbiter Name'],
+    [eventName, seteventName, 'Event Name'],
+  ];
+
+  const validateArbiterInfo = (arbiterInfo: ArbiterInfo): boolean => {
+    if (arbiterInfo.arbiterToken == '') {
+      showError('Arbiter Email token cannot be blank');
+      return false;
+    }
+    if (arbiterInfo.divisionId == '') {
+      showError('Division Id cannot be blank');
+      return false;
+    }
+    if (arbiterInfo.eventName == '') {
+      showError('Event Name cannot be blank');
+      return false;
+    }
+    if (arbiterInfo.userName == '') {
+      showError('Arbiter Name cannot be blank');
+      return false;
+    }
+    if (arbiterInfo.userId == '') {
+      showError('Arbiter Id cannot be blank');
+      return false;
+    }
+    if (arbiterInfo.pin == '') {
+      showError('Arbiter Pin cannot be blank');
+      return false;
+    }
+    if (!validUrl(arbiterInfo.broadcastUrl)) {
+      showError('Invalid Pgn Url');
+      return false;
+    }
+    return true;
+  };
+
+  const confirmArbiterInfo = async (): Promise<void> => {
+    const newArbiterInfo: ArbiterInfo = {
+      pin: arbiterPin.trim().replace(/^\s+|\s+$/g, ''),
+      broadcastUrl: pgnUrl.trim().replace(/^\s+|\s+$/g, ''),
+      arbiterToken: arbiterEmailSecret.trim().replace(/^\s+|\s+$/g, ''),
+      divisionId: divisionId.trim().replace(/^\s+|\s+$/g, ''),
+      userId: arbiterId.trim().replace(/^\s+|\s+$/g, ''),
+      userName: arbiterName.trim().replace(/^\s+|\s+$/g, ''),
+      eventName: eventName.trim().replace(/^\s+|\s+$/g, ''),
+    };
+    dismiss();
+
+    if (validateArbiterInfo(newArbiterInfo)) {
+      await setArbiterInfo(newArbiterInfo);
+    }
+  };
+  return (
+    <>
+      <Sheet title="Edit Event" dismiss={dismiss} visible={display}>
+        <View
+          style={{ display: 'flex', flexDirection: 'row', paddingBottom: 20 }}>
+          <IconButton
+            size={60}
+            icon={inScanQRMode ? 'title' : 'qr-code-scanner'}
+            onPress={() => setInScanQRMode(!inScanQRMode)}
+            colour="black"
+            style={styles.switchInputModeButton}
+          />
+          <PrimaryText
+            size={26}
+            weight={FontWeight.Medium}
+            style={styles.instructions}>
+            Go to{' '}
+            <Link
+              style={styles.instructionsLink}
+              label="tornelo.com"
+              link={torneloUrl}
+            />{' '}
+            and either generate a QR code or manually edit the fields here
+          </PrimaryText>
+        </View>
+        {/* QR Code Entry */}
+        {inScanQRMode && (
+          <View style={styles.editEventSheetSection}>
+            {/* Placeholder - QR Code scanning camera section will go here call confirmArbiterInfo() once the qr code has scanned and everything will be saved!*/}
+            <View
+              style={{
+                backgroundColor: colours.grey10,
+                padding: 30,
```

Comment:
Again here, should the style information be in style.ts?

Reply:
This is a placeholder and will be replaced once bryn implements the qr code stuff, so i didnt bother